tests/bsim/bluetooth/mesh/src/test_suspend.c

```diff
@@ -290,18 +425,68 @@ static void test_rx_suspend_disable_resume(void)
 	PASS();
 }
 
+static void test_dut_gatt_suspend_resume(void)
+{
+	dut_gatt_body(false);
+	PASS();
+}
+
+static void test_dut_gatt_suspend_disable_resume(void)
+{
+	dut_gatt_body(true);
+	PASS();
+}
+
+static void test_tester_gatt(void)
+{
+	k_sem_init(&publish_sem, 0, 1);
+	dut_status = DUT_RUNNING;
+
+	ASSERT_OK_MSG(bt_enable(NULL), "Bluetooth init failed");
+	LOG_INF("Bluetooth initialized");
+
+	/* Scan pb gatt beacons. */
+	gatt_rx_beacons();
+
+	/* Delay to provision DUT */
+	k_sleep(K_MSEC(1000));
+
+	/* Scan gatt proxy beacons. */
+	gatt_param.transmits = 5000 / 1000;
```

Comment:
I would add/copy the `/* (total transmit duration) / (transmit interval) */` line from `test_advertiser.c` as well. It makes it easier to understand why it is written in such a way😺

Reply:
Donezo!